fronts-client/src/components/form/BoostToggles.tsx

```diff
@@ -0,0 +1,109 @@
+import InputRadio from 'components/inputs/InputRadio';
+import React from 'react';
+import { Field } from 'redux-form';
+import type { BoostLevels } from 'types/Collection';
+
+/**
+ * This function generates a set of radio button toggles for selecting a card's "boost level"
+ * in a flexible collection. Boost levels help establish a visual hierarchy within a collection.
+ *
+ * ## Boost Levels:
+ * - **default**: No boost
+ * - **boost**: Slight emphasis
+ * - **megaBoost**: Stronger emphasis
+ * - **gigaBoost**: Most prominent
+ *
+ */
+
+type Toggle = { id: string; value: BoostLevels; label: string };
+
+// Define all possible boost levels for toggles
+const TOGGLES: Record<BoostLevels, Toggle> = {
+	default: {
+		id: 'boostlevel-0',
+		value: 'default',
+		label: 'default',
+	},
+	boost: {
+		id: 'boostlevel-1',
+		value: 'boost',
+		label: 'boost',
+	},
+	megaboost: {
+		id: 'boostlevel-2',
+		value: 'megaboost',
+		label: 'megaboost',
+	},
+	gigaboost: {
+		id: 'boostlevel-3',
+		value: 'gigaboost',
+		label: 'gigaboost',
+	},
+};
+
+/**
+ * ## Presets:
+ * To simplify the boosting process for editors, we provide preset groups for different collection types.
+ * We override the labels in some groups so they make more sense in the context of the collection.
+ *
+ * ### Flexible General (4 Groups):
+ * - **Group 0 (Standard)**: Default, Boost, MegaBoost
+ * - **Group 1 (Big)**: Boost, MegaBoost
+ * - **Group 2 (Very Big)**: MegaBoost
+ * - **Group 3 (Splash)**: Default, Boost, MegaBoost, GigaBoost
+ *
+ * ### Flexible Special (2 Groups):
+ * - **Group 0 (Standard)**: Default, Boost, MegaBoost
+ * - **Group 1 (Splash)**: Default, Boost, MegaBoost, GigaBoost
+ *
+ * */
+const CollectionToggles: Record<string, Record<number, Toggle[]>> = {
+	'flexible/general': {
+		0: [TOGGLES.default, TOGGLES.boost, TOGGLES.megaboost],
+		1: [
+			{ ...TOGGLES.boost, label: 'default' },
+			{ ...TOGGLES.megaboost, label: 'boost' },
+		],
+		2: [{ ...TOGGLES.megaboost, label: 'default' }],
```

Comment:
Given there is only one option here, we might want to change this to an empty array so that nothing renders rather than an unchangeable "default" radio button. 

We will need feedback from editorial stakeholders before making this change so for now i have left this toggle in and will fix forward if required.

Reply:
This has been discussed and editorial would like to keep the default toggle visible.